src/request/change-password/index.html

```diff
@@ -0,0 +1,175 @@
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-eval';
+          img-src http: https: blob: data:;
+          child-src 'self' blob:;
+          style-src 'self' https://fonts.googleapis.com;
+          font-src https://fonts.gstatic.com;">
+    <title>Nimiq Keyguard</title>
+
+    <script defer src="../../../node_modules/@nimiq/core-web/web-offline.js"></script>
+
+    <script defer bundle-common src="../../lib/JsonUtils.js"></script>
+    <script defer bundle-common src="../../lib/RpcServer.es.js"></script>
+    <script defer bundle-common src="../../lib/KeyStore.js"></script>
+    <script defer bundle-common src="../../lib/Key.js"></script>
+    <script defer bundle-common src="../../lib/KeyInfo.js"></script>
+    <script defer bundle-common src="../../lib/Errors.js"></script>
+    <script defer bundle-common src="../../lib/ErrorConstants.js"></script>
+    <script defer bundle-common src="../../lib/Constants.js"></script>
+    <script defer bundle-common src="../../lib/CookieJar.js"></script>
+    <script defer bundle-common src="../../lib/BrowserDetection.js"></script>
+    <script defer bundle-common src="../../lib/AccountStore.js"></script>
+    <script defer bundle-common src="../../common.js"></script>
+
+    <script defer bundle-toplevel src="../../components/NoReferrerErrorPage.js"></script>
+    <script defer bundle-toplevel src="../../components/PassphraseBox.js"></script>
+    <script defer bundle-toplevel src="../../components/PassphraseInput.js"></script>
+    <script defer bundle-toplevel src="../../lib/AnimationUtils.js"></script>
+    <script defer bundle-toplevel src="../../lib/I18n.js"></script>
+    <script defer bundle-toplevel src="../../lib/RequestParser.js"></script>
+    <script defer bundle-toplevel src="../../lib/Utf8Tools.js"></script>
+    <script defer bundle-toplevel src="../TopLevelApi.js"></script>
+    <script defer bundle-toplevel src="../../translations/index.js"></script>
+
+    <script defer src="../../lib/Iqons.js"></script>
+    <script defer src="../../lib/LoginFile.js"></script>
+    <script defer src="../../lib/PasswordStrength.js"></script>
+    <script defer src="../../lib/QrEncoder.js"></script>
+    <script defer src="../../components/DownloadLoginFile.js"></script>
+    <script defer src="../../components/LoginFileIcon.js"></script>
+    <script defer src="../../components/ProgressIndicator.js"></script>
+    <script defer src="../../components/PassphraseSetterBox.js"></script>
+    <script defer src="ChangePassword.js"></script>
+    <script defer src="ChangePasswordApi.js"></script>
+    <script defer src="index.js"></script>
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Muli:400,600,700">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css?family=Fira+Mono&text=0123456789ABCDEFGHJKLMNPQRSTUVXY">
```

Comment:
There's really no need to have this in two lines, because no other index.html does it (the monospace font might even be unnecessary in this request? Is it even during change-password?)

Reply:
Sorry, for some reason my VS Code thinks it's a good idea.
You're right, no address, no Fira. I'll remove it.